tests/Argu.Tests/Tests.fs

```diff
@@ -852,6 +867,11 @@ module ``Argu Tests Main List`` =
         let results = parser.ParseCommandLine (args, raiseOnUsage = false)
         test <@ results.IsUsageRequested @>
 
+    [<Fact>]
+    let ``Required subcommand attribute should fail on missing subcommand and display usage of subcommand and not main command`` () =
+        let args = [|"--mandatory-arg" ; "true" ;  "multiple-mandatories" ; "--valuea"; "5"|]
+        raisesWith<ArguParseException> <@ parser.ParseCommandLine(args) @>
+                                        (fun e -> <@ e.Message.Contains $"USAGE: {parser.ProgramName} multiple-mandatories [--help] --valuea <int>" @>)
```

Comment:
nice use of ProgramName!
while L490 might also cover it, it's some distance away, so I think adding `&& <something that checks that valueb, valuec are reported missing` might be a good idea?

Reply:
if it improves confidence in testing, it's a good idea :)
done

Reply:
yeah, but you're still right to consider duplication/DRY in tests
even if an actual assertion clause is not the solution, sometimes a quick comment can help
you have to remember that you've trawled through the code and some things have become obvious to you in a way that a randomer (like me!) walking the tests to infer usage/features will not!